### PR TITLE
Lecture 03-a

### PR DESCRIPTION
**Pull request by nickgreenquist**
Neural networks can be used for non-linear classification. By warping the input space and introducing non-linearities to our model, we can acheive high accuracy on difficult classification tasks. We also begin to see the inspiration for Convolutional Networks. By sharing parameters in the network, we can use the technique of 'sliding' our network accross different portions of the input. Importantly, training of this type of network is still able to be done using backpropogation and gradient descent. 

-- First part of Lecture 3 (11 February 2019) by Fekade Brook, Nicholas Greenquist, Aaron Wong

### CODE REVIEW

**Comment by Atcold** [comment]
Well done, thanks.

**Comment by Atcold** [comment]
Maybe I've missed it, while skimming your write up.
Did you mention the initial value of the loss and ow to compute it?

**Comment by nickgreenquist** [comment]
@Atcold What we have on that is this:

'Remember, with 3 classes, random guessing would give us 33.33% accuracy so we are still doingsomewhat better than chance. This is because the model can do some work by simply rotating thedecision boundaries.
**NB**: Always print the initial loss before any optimizer steps. This is because we need to see ifour model is actually improving at all from a random initialization of weights.'

Let me know if you want us to mention how to compute it (something like : before taking any optimizaiton steps, compute the loss function using the initial, untouched, theta)

Note: some of the words get squished from copying from Overleaf for some reason

**Comment by Atcold** [comment]
Hmm, yes, the analytical computation of the initial loss is missing. Remember when I've asked to use a calculator to compute `log(3)`?

**Comment by nickgreenquist** [comment]
Im on it, Ill add some notes regarding that discussion in lecture.

**Comment by Atcold** [comment]
And then send a new PR.
Don't forget to pull from `master`, since I've changed a few things.

**Comment by nickgreenquist** [comment]
@Atcold Here is the new PR: https://github.com/Atcold/SP19-DL-collaborative-notes/pull/12

I forgot to pull from master before I made the changes, however, even tho the branch is behind, there are no conflicts with that local change in the file.

**Comment by nickgreenquist** [comment]
@Atcold It should merge smoothly. If not, I can pull from master and get a fresh PR going
